docs/specification/current.md

```diff
@@ -350,7 +350,8 @@ Only the `encounterId` field in this example is eligible to be a prefetch token
   "prefetch": {
     "patient": "Patient/{{context.patientId}}",
     "hemoglobin-a1c": "Observation?patient={{context.patientId}}&code=4548-4&_count=1&sort:desc=date",
-    "user": "{{context.userId}}"
+    "diabetes-type2": "Condition?patient={{context.patientId}}&code=44054006&category=problem-list-item&status=active",
+    "user": "{{userPractitionerRoleId}}"
```

Comment:
This user query change isn't valid since prefetch requirements must be FHIR queries.

Reply:
```suggestion
    "user": "PractitionerRole?_id={{userPractitionerRoleId}}"
```